api/v1/helper.go

```diff
@@ -722,7 +722,12 @@ func (cr *SriovIBNetwork) RenderNetAttDef() (*uns.Unstructured, error) {
 		data.Data["CapabilitiesConfigured"] = true
 		data.Data["SriovCniCapabilities"] = cr.Spec.Capabilities
 	}
-
+	if cr.Spec.PKey == "" {
+		data.Data["pKeyConfigured"] = false
```

Comment:
Hmm why do we need this boolean? Can't we just check for a non-empty `pKey` string field?

Reply:
Tried to do that - didn't seem that simple, so followed the pattern I saw here (e.g `CapabilitiesConfigured`, `LogLevelConfigured`, `LogFileConfigured` etc.